latexify/core.py

```diff
@@ -3,9 +3,31 @@
 import ast
 import math
 import inspect
+from typing import Callable
 
 
 class LatexifyVisitor(ast.NodeVisitor):
+
+  def __init__(self, math_symbol=True):
+    self.math_symbol = math_symbol
+    super(ast.NodeVisitor).__init__()
+
+  def _parse_math_symbols(self, val: str) -> str:
+    if not self.math_symbol:
+      return val
+    greek_and_hebrew = [
+        'aleph', 'alpha', 'beta', 'beth', 'chi', 'daleth',
+        'delta', 'digamma', 'epsilon', 'eta', 'gamma', 'gimel',
```

Comment:
`gamma` hurts `math.gamma` and it needs some workaround (just replacing `math.gamma` with `math.{\\gamma}`?).